components/automate-ui/src/app/modules/infra-proxy/org-details/org-details.component.ts

```diff
@@ -103,6 +111,9 @@ export class OrgDetailsComponent implements OnInit, OnDestroy {
         this.telemetryService.track('orgDetailsTab', 'clients');
         break;
       case 5:
+        this.telemetryService.track('orgDetailsTab', 'policyFiles');
+        break;
+      case 6:
         this.telemetryService.track('orgDetailsTab', 'org-edit');
```

Comment:
[Minor] Seems like most of these are camel case but this one ('org-edit') is kebab case. Could we make that 'orgEdit' for consistency perhaps...?